### DISCUSSION TITLE
Simple 2/3/4 button remote?

### DISCUSSION THREAD

**Discussion by pyrodex** [merbanan/rtl_433#2676] [Q&A]
Does anyone know of any working products in both in the 433 mhz or 915 mhz frequencies similar footprint or style like the image below?

![image](https://github.com/merbanan/rtl_433/assets/108703/db73e61d-c1fa-4a67-ba1a-1e1376676c99)

Looking to slap something on the side of our nightstands and leverage them with functions.

Thanks!

**Comment by zuckschwerdt** [comment]
https://sonoff.tech/product/accessories/t2eu-rf/
or at bit more universal, but it's actually a remote:
https://sonoff.tech/product/accessories/rm433/
